File: code/1010/1010.js
const input = require("fs").readFileSync("/dev/stdin", "utf8");
const lines = input.split(/\s|\n/);

const quant1 = parseInt(lines[1]);
const quant2 = parseInt(lines[4]);

const value1 = parseFloat(lines[2]);
const value2 = parseFloat(lines[5]);

console.log(
  `VALOR A PAGAR: R\$ ${(quant1 * value1 + quant2 * value2).toFixed(2)}`
);

File: code/1010/1010.py
input_line_one = input().split(" ")
input_line_two = input().split(" ")

product_one_quantity = int(input_line_one[1])
product_one_price = float(input_line_one[2])

product_two_quantity = int(input_line_two[1])
product_two_price = float(input_line_two[2])

amount_to_pay = (product_one_quantity * product_one_price) + (
    product_two_quantity * product_two_price
)

print("VALOR A PAGAR: R$ %.2f" % amount_to_pay)
